src/Chart/BarChart.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
```

Comment:
```suggestion
"use client";

import React from "react";
```  

This component is not RSC ready

Reply:
You are right!